<launch>

    <arg name="pressure_header_frame_id_arg" default="odom"/>
    <arg name="pressure_child_frame_id_arg" default="base_link"/>

    <arg name="atm_pressure_arg" default="14.6959"/>
    <arg name="depth_offset_arg" default="0"/>

    <include file="$(find zeabus_elec_ros_power_dist)/launch/power_dist.launch"/>

    <include file="$(find zeabus_elec_ros_peripheral_bridge)/launch/peripheral_bridge.launch"/>

    <node name="Zeabus_Elec_Hardware_interface" pkg="zeabus_elec_ros_hardware_interface" type="zeabus_elec_ros_hardware_interface">

        <param name="pressure_header_frame_id" value="$(arg pressure_header_frame_id_arg)"/>
        <param name="pressure_child_frame_id" value="$(arg pressure_child_frame_id_arg)"/>

        <param name="atm_pressure" value="$(arg atm_pressure_arg)"/>
        <param name="depth_offset" value="$(arg depth_offset_arg)"/>

    </node>

</launch>

